symengine/tests/test_sympify.py

```diff
@@ -14,6 +15,32 @@ def test_sympify1():
     assert false == sympify(False)
 
 
+def test_S():
+    assert S(0) == Integer(0)
+    assert S(1) == Integer(1)
+    assert S(-1) == Integer(-1)
+    assert S(1)/2 == Rational(1, 2)
+    assert S.One is S(1)
+    assert S.Zero is S(0)
+    assert S.NegativeOne is S(-1)
+    assert S.Half is S(1)/2
+    assert S.Pi is pi
+    assert S.NaN is nan
+    assert S.Infinity is oo
+    assert S.NegativeInfinity is -oo
+    assert S.ComplexInfinity is zoo
+    assert S.Exp1 is E
+    assert S.ImaginaryUnit is I
```

Comment:
Please use some way to get these expressions. For eg. `sqrt(-1) is S.ImaginaryUnit`. otherwise there's no point in these tests.